google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -439,19 +434,66 @@ def immediate_future(object: Any) -> concurrent.futures.Future:
         fut.set_result(object)
         return fut
 
-    def connect(self,) -> OpenSSL.SSL.Connection:
-        """A method that returns an OpenSSL connection to the database.
+    def connect(self, driver: str, **kwargs) -> Any:
+        """A method that returns a DB-API connection to the database.
 
-        :rtype: OpenSSl.SSL.Connection
-        :returns: An OpenSSL connection to the primary IP of the database.
+        :type driver: str
+        :param driver: A string representing the driver. e.g. "pg8000" or "pymysql"
```

Comment:
It seems the pg8000 support is gone now?

Reply:
Yeah, still need to work out the specifics on the postgres drivers